xml/admin_install_salt.xml

```diff
@@ -773,6 +773,17 @@ alternative_defaults:
      before &deepsea; stages. To migrate existing directories or for more
      details, see <xref linkend="storage-tips-ceph-btrfs-subvol"/>.
     </para>
+    <para>Apply the following commands to each of the &salt; minions:</para>
+<screen>&prompt.smaster;salt 'MONITOR_NODES' saltutil.sync_all
+&prompt.smaster;salt 'MONITOR_NODES' state.apply ceph.subvolume
+</screen>
+    <note>
+      <para>The &ceph;.subvolume command creates <filename>/var/lib/ceph</filename>
+      as a <filename>@/var/lib/ceph</filename> Btrfs subvolume.
+      </para>
+    </note>
+    <para>Mount the new subvolume and update <filename>/etc/fstab</filename>
```

Comment:
Manually mounting isn't necessary.  The ceph.subvolume state does that as well.

Reply:
Suggested change:

"The new subvolume is now mounted. Update `/etc/fstab` appropriately:"